zebrad/src/commands/generate.rs

```diff
@@ -19,23 +19,23 @@ impl Runnable for GenerateCmd {
 #
 # This file can be used as a skeleton for custom configs.
 #
-# This file is generated using zebrad's current defaults. If you want zebrad
-# to automatically use any newer defaults, set the config options you want to
-# keep, and delete the rest.
+# Unspecified fields use default values. Optional fields are Some(field) if the
+# field is present and None if it is absent.
 #
-# The config format is documented here:
+# This file is generated as an example using zebrad's current defaults.
+# You should set only the config options you want to keep, and delete the rest.
+# Only a subset of fields are present in the skeleton, since optional values
+# whose default is None are omitted.
+#
+# The config format (including a complete list of sections and fields) is
+# documented here:
 # https://doc.zebra.zfnd.org/zebrad/config/struct.ZebradConfig.html
-
-# Usage:
-#     zebrad generate -o myzebrad.toml
```

Comment:
removed instructions on how to run the command from the command's output -- if you got this far, you've already generated a config

Reply:
That's not always true - you might have got your config from someone else, generated it earlier and forgotten how, or got it from a software distribution.

Reply:
Good point, I'll set this back.